src/client/src/rt-platforms/openFin/adapter/window.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-undef */
-import { WindowConfig } from '../../types'
```

Comment:
Do we have listing settings that control spacing around braces (since it is changing)?

Reply:
it's my intelij config. will changem thanks for pointing out